buildpacks/sbt/src/main.rs

```diff
@@ -102,24 +103,30 @@ impl Buildpack for SbtBuildpack {
         handle_sbt_boot(&context, sbt_version, sbt_available_at_launch, &mut env)?;
         handle_sbt_global(&context, sbt_available_at_launch, &mut env)?;
 
-        log_header("Building Scala project");
-
         let tasks = sbt::tasks::from_config(&buildpack_configuration);
-        log_info(format!("Running: sbt {}", shell_words::join(&tasks)));
-
-        let output = Command::new("sbt")
-            .current_dir(&context.app_dir)
-            .args(tasks)
-            .envs(&env)
-            .output_and_write_streams(stdout(), stderr())
-            .map_err(SbtBuildpackError::SbtBuildIoError)?;
-
-        output.status.success().then_some(()).ok_or(
-            SbtBuildpackError::SbtBuildUnexpectedExitStatus(
-                output.status,
-                sbt::output::parse_errors(&output.stdout),
-            ),
-        )?;
+
+        output::track_timing(|| {
+            output::print_section("Running sbt build");
+            output::print_subsection(BuildpackOutputText::new(vec![
+                BuildpackOutputTextSection::regular("Running "),
+                BuildpackOutputTextSection::value(format!("sbt {}", shell_words::join(&tasks))),
+            ]));
```

Comment:
You're printing exactly what you're going to run here (yay!). I still request to refactor based off of a single source of truth such that someone changing the command arguments won't accidentally diverge from what's being printed.

Reply:
I moved this feedback to a "lster" issue https://github.com/heroku/buildpacks-jvm/issues/762